guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -101,34 +101,34 @@ sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
 sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
 sslverify = 1
 
-[{RepoRHEL7Server}]
-name=Red Hat Enterprise Linux 7 Server RPMs x86_64
+[{RepoRHEL8BaseOS}]
+name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
 baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/os/
```

Comment:
this URL is not right (but also not only for the rhel7 vs rhel8 reason), I'd have to look up how the repos now look like

Reply:
@evgeni let me know if you know how these repos look like now and how I should change this.

Reply:
```suggestion
baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel8/$releasever/x86_64/baseos/os
```